src/main/java/JavaReferenceDataTypes/ReWriteMethod.java

```diff
@@ -0,0 +1,41 @@
+package JavaReferenceDataTypes;
+
+public class ReWriteMethod {
+    public static void main(String[] args) {
+        System.out.println(getResult(new int[]{7, 1, 2, 4, 6}));
+        System.out.println(findIndexOfNumberInArray(new int[]{2, 5, 9, 7, 6}, 4));
+        System.out.println(countNumberInArray(new int[]{2, 3, 4, 5, 6, 7, 8}, 6));
+
```

Comment:
Try to avoid leaving empty spaces inside methods.

Reply:
Done